main.tf

```diff
@@ -11,6 +11,7 @@ locals {
 
   additional_tags = {
     autoShutdown = var.autoShutdown
+    startupMode  = var.startupMode
```

Comment:
Is this what we want?

I think @timja mentioned on standup a new value for `autoShutdown` likes of "ondemand"

Reply:
Resolving, another discussion happened where we want to introduce another tag.